TRB_label_assist/generate_figs_for_poster.ipynb

```diff
@@ -29,15 +29,15 @@
                 "import numpy as np\n",
                 "import matplotlib.pyplot as plt\n",
                 "import matplotlib\n",
-                "\n",
+                "import itertools\n",
```

Comment:
Ditto

Reply:
done. This was not tested till that point.